microprofile.ls/com.redhat.microprofile.ls/src/main/resources/com/redhat/microprofile/snippets/mp-openapi.json

```diff
@@ -93,7 +93,7 @@
     ],
     "description": "Encapsulates input parameters.",
     "context": {
-      "type": "org.eclipse.microprofile.openapi.annotations.Parameters"
+      "type": "org.eclipse.microprofile.openapi.annotations.parameters.Parameters.Parameters"

```

Comment:
I think this should just be `org.eclipse.microprofile.openapi.annotations.parameters.Parameters`

Reply:
fixed